examples/Account/Account.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+﻿BeforeAll {
+  Import-Module Az.InfrastructureTesting
+}
+
+Describe 'Verify Account' {
+  It "Should connected to Azure" {
+    # act and assert
+    Confirm-AzBPAccount | Should -BeSuccessful
+  }
+
+  It "Should not contain an Action Group named $noActionGroupName" {
```

Comment:
The name of the test doesn't reflect what is happening test. Can it be adjusted? Also consider moving this to the public tests

Reply:
examples folder removed completely 👍